assets/js/builder/Views/SettingsFields.js

```diff
@@ -352,6 +353,10 @@ define( [], function() {
 					defaults.input_type  = 'url';
 				break;
 
+				case 'disabled':
+					defaults.url = 'https://lifterlms.com/product/advanced-quizzes/?utm_source=LifterLMS%20Plugin&utm_medium=Quiz%20Builder%20Button&utm_campaign=Advanced%20Question%20Upsell&utm_content=3.16.0&utm_term=Questions%20Bank';
```

Comment:
we're creating a type here with a default which applies to only one case, it's too specific, it's not a default.